src/neo/IO/Helper.cs

```diff
@@ -162,9 +162,20 @@ public static byte[] ReadBytesWithGrouping(this BinaryReader reader)
             }
         }
 
+        public static byte[] ReadFixedBytes(this BinaryReader reader, int size)
+        {
+            var data = new byte[size];
```

Comment:
This is the old implementation, but we can change this to allow to read from buffered stream like TcpStream, that it doesn't ensure the complete read at once